hive_metastore_client/hive_metastore_client.py

```diff
@@ -329,16 +329,19 @@ def get_partition_values_from_table(
         :param db_name: database name where the table is at
         :param table_name: table name which the partitions belong to
         """
-        partition_values_response = self.get_partition_values(
-            PartitionValuesRequest(
-                dbName=db_name,
-                tblName=table_name,
-                partitionKeys=self.get_partition_keys_objects(
-                    db_name=db_name, table_name=table_name
-                ),
+        try:
+            partition_values_response = self.get_partition_values(
+                PartitionValuesRequest(
+                    dbName=db_name,
+                    tblName=table_name,
+                    partitionKeys=self.get_partition_keys_objects(
+                        db_name=db_name, table_name=table_name
+                    ),
+                )
             )
-        )
-
-        return [
-            partition.row for partition in partition_values_response.partitionValues
-        ]
+            partitions = [
+                partition.row for partition in partition_values_response.partitionValues
+            ]
+        except Exception:
```

Comment:
We just need to keep in mind that this could open the door for other exceptions to be silently passing. We can keep an eye on it and refactor if it happens :)

Reply:
sure!
Now the exception raised is also too broad. The exception raised when reading partition non-partitioned tables is a TTransportException, raised in many contexts. So there isn't much we can do here too :/